test_api.py

```diff
@@ -0,0 +1,61 @@
+import pytest
```

Comment:
Put all test related stuff in a `tests` folder

Reply:
Sure, I'll move all the test-related files into a `tests` folder. Thank you for the feedback!